lib/validation.rb

```diff
@@ -0,0 +1,21 @@
+module Validation
+  def ensure_format(key)
+    if key =~ %r!\A(GET|PUT|POST|DELETE|HEAD)\s\/[a-z0-9_]+\z!
```

Comment:
__[rubocop]__
* Style/PercentLiteralDelimiters: `%r`-literals should be delimited by `{` and `}`.